requirements.txt

```diff
@@ -0,0 +1,10 @@
+numpy
+scipy
```

Comment:
If I rebuild my environment with these changes and inspect whether my numpy and scipy come from pip or conda I get that they come from pip...

```shell
$conda list | grep "numpy\|scipy"

numpy                     1.21.4                   pypi_0    pypi
scipy                     1.7.2                    pypi_0    pypi
```

(I think) because e.g. `pandas` in `requirements.txt` will get those again from PyPi


Can you confirm the same result?

Reply:
I think what might need adding to `environment.yaml` to make `scipy` work for you is `openblas` or `mkl`.

Reply:
Mine look like they've been installed from conda

```bash
$conda list | grep "numpy\|scipy"

numpy                     1.21.1           py38h1a24bff_2
scipy                     1.7.1            py38h2f0f56f_2
```

even though
```bash
$conda list | grep "pandas"

pandas                    1.3.4                    pypi_0    pypi
```

I also tested out
```bash
$conda create -n test_env python=3.8 openblas
$conda activate test_env
$pip install scipy
```

which ran fine and seemed to work when testing in ipython so seems like a good fix